driver-core/src/main/java/com/datastax/driver/core/Metrics.java

```diff
@@ -219,6 +229,15 @@ public Errors getErrorMetrics() {
     }
 
     /**
+     * Returns the total number of in flight requests to Cassandra hosts.
+     *
+     * @return The total number of in flight requests to Cassandra hosts.
+     */
+    public Gauge<Integer> getInFlightRequests() {
```

Comment:
Isn't this a bit too coarse-grained? I wonder if it would be better to return a `MetricSet` instead, and return the number of inflight requests on a per-host basis. This would allow users to detect slow hosts (and besides, this is exactly what has been suggested recently as the best available metric to measure host performance – much better than request latencies for instance).